.coreos/desoto.service

```diff
@@ -9,23 +9,13 @@ Restart=always
 TimeoutStartSec=10 min
 User=core
 KillMode=none
-
 EnvironmentFile=/etc/environment
-
-ExecStartPre=/usr/bin/env bash -c "etcdctl get /environments/%p > '/tmp/%p.env'"
-
+ExecStartPre=/usr/bin/env bash -c "etcdctl --no-sync get /environments/%p > '/tmp/%p.env' && echo \"ETCD_HOSTS=http://`ifconfig docker0 | awk '/\<inet\>/ { print $2}'`:4001\" >> '/tmp/%p.env'"
 ExecStartPre=-/usr/bin/env docker stop "%p"
 ExecStartPre=-/usr/bin/env docker rm "%p"
 ExecStartPre=/usr/bin/env docker pull christianbladescb/desoto
```

Comment:
I believe desoto should be hosted on Quay and all uses of `christianbladescb/desoto` be replaced.